blog/secadvisor.md

```diff
@@ -0,0 +1,363 @@
+In the secadvisor.yml file, uncomment the iam_endpoint field and set it to https://iam.cloud.ibm.com/identity/token.
+
+### Setup Minio
+
+For running tests on a local machine, it is possible to set up a local Minio object store.
+
+For details, see the instructions found at [Secadvisor Developer's README](https://github.com/skydive-project/skydive/tree/master/contrib/pipelines/secadvisor/README.md).
+
+## Deploy the Security Advisor pipeline
+
+Build and run the pipeline in the secadvisor directory: $SKYDIVE_BASE/contrib/pipelines/secadvisor.
+
+```
+cd $SKYDIVE_BASE/contrib/pipelines/secadvisor
+make static
+```
+
+The binary is created in the same directory: $SKYDIVE_BASE/contrib/pipelines/secadvisor/secadvisor.
+
+### Activate the Security Advisor
+
+```
+./secadvisor /etc/skydive/secadvisor.yml
+```
+
+Note that it is possible to run several instances of the Security Advisor at the same time, each one with a different configuration file.
+This is useful to capture different flows for different purposes and to perform different filtering (e.g. based on specified cluster_net_masks and filter parameters).
+
+## Generate and Capture Flows
+
+### Capture Flows
+
+Via the Skydive WebUI setup captures and generate traffic which should
+result in the secadvisor pipeline sending flows to the ObjectStore.
+
+To connect to the GUI, open a web browzer to the address of the Skydive analyzer at port 8082. If running the browzer on the same machine as the analyzer, then connect to localhost:8082.
+You should see the topology of your network, perhaps something like the following image.
+
+<p class="center">
+  <a href="/assets/images/blog/capture-future-1.png" data-lightbox="Skydive Topology" data-title="Skydive topology view">
+    <img src="/assets/images/blog/capture-future-1.png"/>
+  </a>
+</p>
+
+
+Make sure you are on the "Captures" view and press on "Create".
+
+Fill in the "Targets" fields by placing the cursor in the first "Interface" field and then use the mouse to point to your network endpoint that you want to capture.
+To capture a particular flow end-to-end, fill in the second "Interface" field in a simmilar manner with the other network endpoint.
+Then press the "Start" button on the GUI.
+
+Alternatively, you can start the capture of flows from the command line with a commond like the following:
+
+```
+skydive client capture create --gremlin "G.V().Has('Type', 'device', 'Name', 'eth0')" --type pcap 
+```
+
+The command specifies to capture all flows that match the gremlin expression - in this case, all devices that have name 'eth0'.
+
+### Generate Flows
+
+Generate some network traffic on the interfaces you specified to capture.
+For example, run some iperf traffic between entities and capture the flow via the Skydive GUI.
+
+Run iperf server:
+
+```
+iperf3 -s
```

Comment:
for completeness please explain howto install iperf3

Reply:
OK